### PR TITLE
Data fetching docs

### PR DESCRIPTION
**Pull request by Vijar**
@redonkulus @jedireza

### CODE REVIEW

File: docs/guides/fetching-data.md [hunk 1]
```diff
@@ -1,3 +1,114 @@
...
+
+On the client, however, we can NOT directly query the database.
+
+Instead, we have to write extra code to:
+ - expose a REST endpoint on the server
+ - make an xhr request from the client
+
+Only then, can the server query the database and return the results back to the client for parsing.
+
+We want to be able to fetch the data on the server and client without extra code.
```
Comment (Vijar) [inline docs/guides/fetching-data.md]: Do we even need the above 10 lines of explanation of the problem?

Comment (redonkulus) [inline docs/guides/fetching-data.md]: I dont think so.

Comment (redonkulus) [inline docs/guides/fetching-data.md]: You could sum it in one sentence by saying about how data fetching is isomorphic with Fetchr.

File: docs/guides/fetching-data.md [hunk 2]
```diff
@@ -1,3 +1,114 @@
...
+
+[Fetchr] is a library that manages an application's services and provides an isomorphic interface for calling the services.
```
Comment (redonkulus) [inline docs/guides/fetching-data.md]: Need link for [Fetchr]

Comment (redonkulus) [inline docs/guides/fetching-data.md]: We mention an applications services, but don't define it til later. What are your thoughts about talking about services first, then mentioning how we use Fetchr to do it isomorphically?

Comment (jedireza) [inline docs/guides/fetching-data.md]: ... a library that manages an application's services (data endpoints) ...

File: docs/guides/fetching-data.md [hunk 3]
```diff
@@ -1,3 +1,114 @@
...
+## Services
+
+Services are where you define your CRUD operations for a specific resource (i.e. Users, Photos, etc.).
+
+### Creating Your Services
+
+UserService.js
+```js
+module.exports = {
+    //Name is required
```
Comment (redonkulus) [inline docs/guides/fetching-data.md]: spaces after comments

File: docs/guides/fetching-data.md (hunk 1 above)
Comment (Vijar) [inline docs/guides/fetching-data.md]: I feel like the following big paragraph does that, so I'll just delete these lines

File: docs/guides/fetching-data.md [hunk 4]
```diff
@@ -1,3 +1,110 @@
-# Fetching data
+# Fetching Data
 
-Details coming soon.
+Data is organized into services.
+Services are where you define your CRUD operations for a specific resource.
```
Comment (redonkulus) [inline docs/guides/fetching-data.md]: Might want to mention what CRUD is. Just simple (create, read, update, delete)

Comment (redonkulus) [inline docs/guides/fetching-data.md]: nevermind its below.

**Comment by redonkulus** [comment]
:shipit:

**Pull request merged**